### ISSUE TITLE
Feature request: Add style file

### ISSUE THREAD

**Issue by borispoehland** [IlusionDev/nextjs-sitemap-generator#38]
Hi, I have a feature request for something that is not that complicated to implement.

You can add a style to your xml file using this line in the xml-file:
`<?xml-stylesheet type="text/xsl" href="http://www.yourdomain.com/sitemap.xsl"?>`

It would be nice to pass the href using the constructor, e.g.

sitemap({
  ...
  stylesheet: `${__dirname}/public/sitemap.xsl`,

})

It would be really nice if you could implement this.
Basically you'd just have to edit core.js:20 with inserting
`<?xml-stylesheet type="text/xsl" href="${stylesheet}"?>`

Best regards

**Comment by IlusionDev** [comment]
I'm going to add it, maybe tomorrow is done!
Thanks for the suggestion.
👋

**Comment by IlusionDev** [comment]
This feature is added. Check it in 0.5.2

**Comment by borispoehland** [comment]
Thank you!

**Comment by borispoehland** [comment]
Hey, I just tried out your solution and unfortunately, it's not working. It works as soon as I put the style-declaration **before the** <urlset>-tag.

**Comment by IlusionDev** [comment]
You can check the new fix at 1.0.0 version

**Comment by borispoehland** [comment]
Works like a charm! Thank you man!!!
